composer.json

```diff
@@ -53,5 +54,8 @@
         }
     },
 
+    "prefer-stable": true,
+    "minimum-stability": "dev",
```

Comment:
Had to it because `puli/composer-plugin` is still in beta (composer doesn't accept to install it then, as it is "not stable" yet)